src/quantum/HISTORY.rst

```diff
@@ -0,0 +1,9 @@
+.. :changelog:
+
+Release History
+===============
+
+0.11.2906.2
```

Comment:
normally version starts from 0.1.0. THis is Azure CLI extension version, no need to sync with service/product version

Reply:
Agreed, however this is going to be released and documented as part of the Quantum Development Kit (QDK), and has specific dependencies on the QDK version so it would greatly help us to keep the versions in-sync. As such, I'd rather keep it like this.

Reply:
Extension version is different from SDK version. So I think it's not a good idea to keep them the same. Besides, CLI only support this version format: {major}.{minor}.{patch}.

Reply:
Run `az extension list-available -otable` to see examples.

Reply:
Hi @qwordy . Since June we've named versions of the CLI using the convention above, which is not following the guidelines. What would you recommend? Should we reset the history to 0.1.0 for our initial release by official means? All the versions listed there are private, and have only been shared with partners. It might be better to restart than to continue with a mix of internal/external versions.

Reply:
I recommend 0.1.0. The version is Azure CLI extension's version. It is not version of quantum service or QDK.